R/checks.R

```diff
@@ -798,14 +798,18 @@ check_fracwith_diswith <- function(form, rep) {
 #' @return a dataframe containing problematic entries with relevant columns
 #' @import tidyverse
 #' @export 
-check_locof1stadmin_pritosur <- function(form) {
+check_ihhrs_ishrs_iahrs_1hrwithin <- function(form) {
   hrswithin <- 1
   problems <- form %>% 
     transmute(
-      region, site, studyid, ptinit, ptstatus, ihhrs, ishrs_1, iahrs,
-      itoshrs = ihhrs + ishrs_1, 
+      region, site, studyid, ptinit, ptstatus, 
```

Comment:
Why coalesce? Let's talk in the meeting

Reply:
I don't totally understand the logic here.

Reply:
You don't need coalesce. If a value is NA, it will simply propagate with arithmetic and logical operations. If a value turns out to be NA, it will not be filtered. You cannot coalesce them to 0. It will include too many entries that are not actionable. We are doing separate checks for NA values.